modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -93,27 +96,7 @@ export default {
         }
       });
       this.icons = icons;
-      // prep row checkbox fields
-    },
-    generateCheckboxes () {
```

Comment:
Only one component (pieces manager) was using this at all. With all the properties of this that are no longer needed it seemed fine to get rid of it. I discussed with Stu as well.